src/cpp/session/modules/SessionPythonEnvironments.R

```diff
@@ -636,9 +636,9 @@
    args <- c("env", "list", "--json", "--quiet")
    tmp <- tempfile()
    output <- system2(conda, args, stdout = TRUE, stderr = tmp)
-   if (attr(output, "status") != 0) {
+   exit_code <- attr(output, "status")
+   if ( !is.null(exit_code) && !identical(exit_code, 0)) {
```

Comment:
I'm not sure if this is correct. For example:

```
> identical(0, 0L)
[1] FALSE
```

And

```
> status <- system("true")
> status
[1] 0
> identical(status, 0L)
[1] TRUE
> identical(status, 0)
[1] FALSE
```

Reply:
Ah okay. Yeah good point I can switch it to `==`. Or possibly just remove it -- AFAIK if you use `system2` (note we're not using system) with stdout = TRUE, it will not actually set status to 0 to indicate success. 
From the help: 
> If stdout = TRUE or stderr = TRUE...If command runs but gives a non-zero exit status this will be reported with a warning and in the attribute "status" of the result
> In other cases _[meaning, when neither stdout nor stderr is TRUE]_, the return value is an error code (0 for success)

Currently merged version is incorrect because attr(output, "status") will be NULL when the command succeeds, and `NULL != 0` is TRUE

Reply:
I see elsewhere in the code we used 

```
status <- .rs.nullCoalesce(attr(result, "status", exact = TRUE), 0L)
   if (!identical(status, 0L))
```
which looks correct so I'm going to change to that